cmd/tests/cmd_run_test.go

```diff
@@ -1827,80 +1827,6 @@ func BenchmarkReadResponseBody(b *testing.B) {
 	cmd.ExecuteWithGlobalState(ts.GlobalState)
 }
 
-func TestBrowserPermissions(t *testing.T) {
```

Comment:
Can we actually have a test with the scenarios options instead?

Reply:
+1 for having an e2e coverage

Reply:
> isn't being closed when the test exits, am i correct in thinking that?

Yes, it seems you're right. Do you have a guess why this is happening? I see the code for unsubscribing is implemented on the core and invoked from the browser.

>I'd prefer to suppress this error for now and work on the fix in the next cycle.

How do you intend to suppress it? Do you want to remove the e2e test? I would prefer if at least we understand the root of the problem and we open an issue so we can classify it as non-critical with confidence.

Reply:
Ok, I'll take a look.

Reply:
I've created a new [issue](https://github.com/grafana/xk6-browser/issues/989). I believe the issue actually lies within the browser module scope and so we will work on it asap.

When the test run completes, does the k6 processes exit, or can it ever be reused? The reason I'm asking is because these uncleared goroutines will leak if the k6 process is reused. If it isn't then this is non-critical issue, but we will not be able to add the e2e test in this release.

Reply:
We have a fix for the leaky goroutines in https://github.com/grafana/xk6-browser/pull/990. As soon as that has been merged into main, i'll upgrade this branch to work with that commit from main, and add the e2e test in.

Reply:
I've updated this branch to work with v1.0.1 of xk6-browser which contains the fix for the goroutine leaks. I've also added an e2e test.

Reply:
[This](https://github.com/grafana/k6/actions/runs/5751162829/job/15589353654?pr=3235) has now been resolved in a32cc6fda7f2f57f8e26a1e0140bb54aa4ceba67. Could you please take a look @mstoykov and @codebien?